src/pages/registration/RegistrationForm.tsx

```diff
@@ -56,7 +56,7 @@ const RegistrationForm = ({ registrationInfo }: RegistrationFormParams) => {
   const isHMDMCEnabled = values.tissues[currentIndex].species === "Human";
   useEffect(() => {
     if (!isHMDMCEnabled) {
-      setFieldValue(`tissues.${currentIndex}.hmdmc`, "", true);
+      setFieldValue(`tissues.${currentIndex}.hmdmc`, "", false);
```

Comment:
Why this change?

Reply:
This is not required. I am reverting it back. Thank you for the finding. 👍